lib/src/facts/solaris/zone_resolver.cc

```diff
@@ -14,7 +14,10 @@ namespace facter { namespace facts { namespace solaris {
     zone_resolver::data zone_resolver::collect_data(collection& facts)
     {
         data result;
-        result.current_zone_name = get<1>(execute("/bin/zonename"));
```

Comment:
This seems cleaner to me (and more consistent with the behavior on failure for the zones fact immediately below).

So I'm :+1: